src/index.html

```diff
@@ -13,6 +13,39 @@
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <section
+      class="product"
+      data-qa="card"
+    >
+      <img
+        class="product__img"
+        src="./images/imac.jpeg"
+        alt="MacBook"
+      />
+      <h1 class="product__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h1>
+      <h2 class="product__code">Product code: 195434</h2>
```

Comment:
example of empty line between sibling blocks. fix the rest :)
```suggestion
      <h1 class="product__title">
        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
      </h1>
      
      <h2 class="product__code">Product code: 195434</h2>
```